src/lib/route-broadcaster.js

```diff
@@ -107,6 +109,31 @@ class RouteBroadcaster {
       (pair) => this._tradingPairToLocalRoute(pair)))
   }
 
+  addConfigRoutes () {
+    let that = this
+
+    return co(function * () {
+      for (let configRoute of that.configRoutes) {
+        const connectorLedger = configRoute.connector
+          .split('.')
+          .slice(0, configRoute.connector.split('.').length - 1)
+          .join('.') + '.'
+
+        const route = new Route(
+          // use a 1:1 curve as a placeholder (it will be overwritten by a remote quote)
+          [ [0, 0], [1, 1] ],
+          // the ledger '.' is inserted to make sure that the hop to the connectorLedger
+          // is not considered final.
+          [ connectorLedger, '.' ],
```

Comment:
This will cause issues when broadcasting routes. The broadcasted route will have a `destinationLedger` of `"."`, which doesn't make sense. Should it be `configRoute.prefix` or similar instead?